src/main/java/com/amazon/sqs/javamessaging/AmazonSQSExtendedClient.java

```diff
@@ -122,13 +124,16 @@ public AmazonSQSExtendedClient(AmazonSQS sqsClient) {
 	 *
 	 * @param sqsClient
 	 *            The Amazon SQS client to use to connect to Amazon SQS.
-	 * @param extendedClientConfig
-	 *            The extended client configuration options controlling the
+	 * @param payloadStorageConfiguration
+	 *            The payload storage configuration options controlling the
 	 *            functionality of this client.
 	 */
-	public AmazonSQSExtendedClient(AmazonSQS sqsClient, ExtendedClientConfiguration extendedClientConfig) {
+	public AmazonSQSExtendedClient(AmazonSQS sqsClient, PayloadStorageConfiguration payloadStorageConfiguration) {
 		super(sqsClient);
-		this.clientConfiguration = new ExtendedClientConfiguration(extendedClientConfig);
+		if (payloadStorageConfiguration.getPayloadSizeThreshold() == 0) {
```

Comment:
Since PayloadStorageConfiguration variable payloadSizeThreshold by default is 0, setting it to SQS default value. Let me know if I should go for another approach.